server/package.json

```diff
@@ -27,17 +28,17 @@
     "passport-jwt": "4.0.0",
     "passport-local": "1.0.0",
     "winston": "3.3.3",
-    "winston-cloudwatch": "0.9.0",
+    "winston-cloudwatch": "3.1.1",
     "yup": "0.32.11"
   },
   "devDependencies": {
-    "eslint": "7.7.0",
+    "eslint": "^7.32.0",
     "eslint-config-airbnb-base": "14.2.0",
     "eslint-plugin-import": "2.22.0",
     "eslint-plugin-jest": "23.20.0",
     "fast-csv": "4.3.6",
     "flat": "5.0.2",
-    "husky": "4.2.5",
+    "husky": "^4.3.8",
```

Comment:
I think `^` was introduced by error?

Reply:
I left it as is since it is a devDependencies and don't mind having the latest version of husky.